tests/test_masking.py

```diff
@@ -14,18 +13,18 @@
 
 
 class TestMasking:
-    def test_load_intervals(self):
+    def test_load_intervals(self, tmp_path):
```

Comment:
```suggestion
    @pytest.usefixtures("tmp_path")
    def test_load_intervals(self, tmp_path):
```
I know this isn't required, but unless one is very familiar with pytest then reading tests with "magic" arguments is very confusing.